index.js

```diff
@@ -24,39 +26,46 @@ module.exports = {
 
     if (!(app.options['materialize-shim'] || {}).omitJS) {
       app.import('vendor/materialize/materialize.js');
-      app.import('vendor/materialize-shim.js', {
-        exports: {
-          materialize: ['default']
-        }
+      app.import('vendor/materialize-shim.js');
+    }
+
+    if (!(app.options['materialize-shim'] || {}).omitFonts) {
+      getFontPaths().robotoFiles.forEach(font => {
+        app.import(`vendor/fonts/roboto/${font}`, {
+          destDir: 'assets'
+        });
       });
     }
   },
 
   treeForVendor(tree) {
-    let trees = [];
+    const trees = [];
 
     if (tree) {
       trees.push(tree);
     }
 
-    let materializePath = path.join(MATERIALIZE_JS_PATH, '..');
-    if (existsSync(materializePath)) {
-      let materializeJsTree = fastbootTransform(
-        new Funnel(materializePath, {
+    if (existsSync(MATERIALIZE_JS_FOLDER)) {
+      const materializeJsTree = fastbootTransform(
+        new Funnel(MATERIALIZE_JS_FOLDER, {
           files: ['materialize.js'],
           destDir: 'materialize'
         })
       );
-      let materializeFontsTree = fastbootTransform(
-        new Funnel(path.join(materializePath, '..', 'fonts'), {
+      trees.push(materializeJsTree);
+    } else {
+      throw new Error(`Could not find materialize JS folder (${MATERIALIZE_JS_FOLDER})`);
+    }
+
+    if (existsSync(FONTS_FOLDER)) {
+      const fontsTree = fastbootTransform(
```

Comment:
I have no idea why we would need `fastbootTransform` for fonts (it only touches, JS, right?), but I left this here because it was there before. Could we safely remove it?